patternfly/patternfly-elements
# WIP Add classes for Typography

## Component name

- pfe-styles -- adding typography classes


### Related issue

- Migrate typography classes to pfe-styles (from redhat-theme)  
 https://github.com/patternfly/patternfly-elements/issues/936


### Preview
 
- [layout](http://localhost:8000/elements/pfe-styles/demo/) 
- [typography](https://deploy-preview-1130--happy-galileo-ea79c4.netlify.app/elements/pfe-styles/demo/typography.html)
- [colors](http://localhost:8000/elements/pfe-styles/demo/colors.html)

### What has changed and why
 
- bring typography classes into PFE from redhat-theme

![image](https://user-images.githubusercontent.com/456863/96497190-4faa2d80-1218-11eb-86be-81f0f3eeaf89.png)

----

### Testing instructions



1. Open the [preview page, typography section](https://deploy-preview-1130--happy-galileo-ea79c4.netlify.app/elements/pfe-styles/demo/#type-with-pfe-classes) 
2. Verify that the heading and text classes are present in the the pfe-type.css, and that they match the design system specs for [desktop & mobile typography](https://www.sketch.com/s/f377ffc2-c43b-4ba6-8504-6396047f96db/a/ZbqM07)


#### Browser requirements

Your component should work in all of the following environments:



### Ready-for-merge Checklist

Check off items as they are completed.  Feel free to delete items if they are not applicable.



### Merging

Please **squash** when merging and ensure your commit message uses [conventional commit](https://www.conventionalcommits.org/en/v1.0.0/#summary) formatting.

**Be sure to share your updates with the [patternfly-elements-contribute@redhat.com](mailto:patternfly-elements-contribute@redhat.com) mailing list!**
